src/app/home/home.component.ts

```diff
@@ -139,16 +139,20 @@ export class HomeComponent implements AfterViewInit {
     }
 
     // TODO data-placeholder broke
-    onTextPaste($event: any) {
+    onTextPaste($event: ClipboardEvent) {
```

Comment:
Can you double check if it is a `ClipboardEvent` regardless of if you paste through the keyboard or through the mouse?

Reply:
Yes it works for both cases. There's no other option for this type of event. Also tested to double check 👍